heroic-component/src/test/java/com/spotify/heroic/aggregation/AggregationOutputTest.java

```diff
@@ -20,16 +20,16 @@ public void lombokDataTest() {
 
     @Test
     public void isEmpty() {
-        final AggregationOutput output =
-            new AggregationOutput(ImmutableMap.of(), ImmutableSet.of(), MetricCollection.empty());
+        final AggregationOutput output = new AggregationOutput(ImmutableMap.of(), ImmutableSet.of(),
+            MetricCollection.points(ImmutableList.of()));
```

Comment:
Say, what's the reason behind removing the EmptyMetricCollection class?

Reply:
Unnecessary null object. It's also mentioned in the description:

> The EmptyMetricCollection null object has been removed. Attempting to share it between nodes results in inconsistent behavior, which is just confusing.

Reply:
Ok I see, thanks. Sorry, didn't see the description note regarding that.